images/hub/jupyterhub_config.py

```diff
@@ -277,12 +277,8 @@ def generate_user_name(spawner):
     if get_config('cull.users'):
         cull_cmd.append('--cull-users')
 
-    # FIXME: remove version check when we require jupyterhub 0.9 in the chart
-    # that will also mean we can remove the podCuller image
-    import jupyterhub
-    from distutils.version import LooseVersion as V
     cull_max_age = get_config('cull.max-age')
-    if cull_max_age and V(jupyterhub.__version__) >= V('0.9'):
+    if cull_max_age:
```

Comment:
This will have the same truth table no matter what, if we assume jupyterhub.__version__ to always be >= 0.9, which we do.